monkey/common/cloud/aws_service.py

```diff
@@ -50,14 +73,22 @@ def test_client():
 
     @staticmethod
     def get_instances():
-        return \
-            [
-                {
-                    'instance_id': x['InstanceId'],
-                    'name': x['ComputerName'],
-                    'os': x['PlatformType'].lower(),
-                    'ip_address': x['IPAddress']
-                }
-                for x in AwsService.get_client('ssm').describe_instance_information()['InstanceInformationList']
-            ]
+        """
+        Get the information for all instances with the relevant roles.
+
+        This function will assume that it's running on an EC2 instance with the correct IAM role.
+        See https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html#iam-role for details.
+
+        :raises: botocore.exceptions.ClientError if can't describe local instance information.
+        :return: All visible instances from this instance
+        """
+        current_instance = AwsInstance()
+        local_ssm_client = boto3.client("ssm", region_name=current_instance.get_region())
```

Comment:
Use `AwsService.get_client('ssm', current_instance.get_region())` for consistency